hphp/util/light-process.cpp

```diff
@@ -436,6 +436,23 @@ void LightProcess::SigChldHandler(int sig, siginfo_t* info, void* ctx) {
   }
 }
 
+void LightProcess::AttachHandler(sighandler_t sighandler)
+{
+  if (sighandler) {
+    always_assert_flog(sighandler == (sighandler_t)&SigChldHandler,
```

Comment:
This seems even more circuitous than before.  If the argument always has to be a constant value, just don't take an argument...